pkg/generate/app/environmentfile.go

```diff
@@ -0,0 +1,24 @@
+package app
+
+import (
+	"bufio"
+	"os"
+)
+
+// GetEnvironmentVars reads a whole file into memory and returns string
+// array. This could be a problem if we are reading a very large file. I am
+// not putting any validation for file, if it contains a key=value pair as
+// that should be handled in environment variables that we are passing.
+func GetEnvironmentVars(path string) ([]string, error) {
```

Comment:
I'm ok with not validating `key=value` here since you are going to hit the validation for `Environment`. But what this function is doing has nothing to do with env vars, it's really just reading the lines from a file. Rename it to something more generic like `ReadLinesFromFile` or `ReadFileLines` and place it somewhere else, like `file.go`.